models.py

```diff
@@ -120,7 +120,7 @@ class SampledBallot(db.Model):
     ballot_position = db.Column(db.Integer, nullable=False)
     
     __table_args__ = (
-        db.PrimaryKeyConstraint('round_id', 'jurisdiction_id', 'batch_id', 'ballot_position'),
+        db.PrimaryKeyConstraint('round_id', 'jurisdiction_id', 'batch_id', 'ballot_position', 'ticket_number'),
```

Comment:
Do you anticipate that there will be `SampledBallot` records with identical `round_id`, `jurisdiction_id`, `batch_id`, and `ballot_position` that only vary by `ticket_number`? If so, I think we'll need to update this endpoint: `/election/<election_id>/jurisdiction/<jurisdiction_id>/batch/<batch_id>/round/<round_id>/ballot/<ballot_position>`.

Reply:
Yeah, I think that is quite likely.

Reply:
@MorganLove do you have a sense of how this would need to change?

Reply:
@benadida can you weigh in on the above? we will definitely need to support selected ballots that differ only by ticket number

Reply:
Ah, I see `API.md` is not up to date. It would be good if we kept it up to date.

As best as I can tell,  `../round/<round_id>/ballot/<ballot_position>` is only for setting a ballot, not fo fetching one, which I think is exactly the right thing, since I don't see any reason to fetch ballots other than by round with `../ballot-list`. The ballot list API call should return all the sampled ballots, with each entry representing a particular ballot position, each time it's sampled, each with its ticket number. When you set the ballot, you only to do that once for that batch ID and position. So I don't think the API needs to change.

I *would* suggest changing the API for setting the ballot from

`POST ../batch/<batch_id>/round/<round_id>/ballot/<ballot_position>`

to

`POST ../batch/<batch_id>/ballot/<ballot_position>`

since the content of a ballot does not depend on the round in which it's selected.

Reply:
> Ah, I see API.md is not up to date. It would be good if we kept it up to date.

*Gazes longingly at tools that generate such documentation automatically.*